packages/server-wallet/src/models/challenge-status.ts

```diff
@@ -0,0 +1,73 @@
+import Knex from 'knex';
+import {Model} from 'objection';
+
+import {Bytes32, Uint48} from '../type-aliases';
+
+export type ChallengeStatusResult =
+  | {status: 'Challenge Finalized'; finalizedAt: number; finalizedBlockNumber: number}
+  | {status: 'Challenge Active'}
```

Comment:
Might we want to include a `finalizesAt` field in the `Challenge Active` type?

Reply:
That probably makes sense. I omitted it for now since we don't care about the `Challenge Active` status.